cli.py

```diff
@@ -68,18 +69,26 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
+@click.option('-j', '--json-file-name', type=str,
+              help="JSON Filename containing info related to simulation")
 @click.option('--run-on-redis', is_flag=True, default=False, help="Start the client using the Redis API")
-def run(setup_module_name, username, password, domain_name, web_socket, run_on_redis, **kwargs):
+def run(setup_module_name, username, password, domain_name, web_socket, json_file_name,
+        run_on_redis, **kwargs):
     if username is not None:
         os.environ["API_CLIENT_USERNAME"] = username
     if password is not None:
         os.environ["API_CLIENT_PASSWORD"] = password
     os.environ["API_CLIENT_DOMAIN_NAME"] = domain_name
     os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"] = web_socket
     os.environ["API_CLIENT_RUN_ON_REDIS"] = "true" if run_on_redis else "false"
+    os.environ["JSON_FILE_PATH"] = os.path.join(modules_path[0], json_file_name)
```

Comment:
This means that the path of the JSON file will always be relative to `modules_path[0]` (that in general is the `setups/` folder). Is this the intended behavior?

Reply:
Will adapt it to make it absolute as well as relative to user's current working dir.